src/common/hook/useQueue/utils/publish.tsx

```diff
@@ -1,9 +1,13 @@
-import { wrapDocuments } from "@govtechsg/open-attestation";
+import {
+  wrapDocuments,
+  __unsafe__use__it__at__your__own__risks__wrapDocuments,
+  utils,
+} from "@govtechsg/open-attestation";
 import { defaultsDeep, groupBy } from "lodash";
 import { identifyProofType } from "../../../../constants/QueueState";
 import { ActionsUrlObject, Config, DocumentStorage, FormEntry, PublishingJob, RawDocument } from "../../../../types";
 import { getQueueNumber } from "../../../API/storageAPI";
-import { encodeQrCode } from "../../../utils";
+import { encodeQrCode, isV3Document } from "../../../utils";
 import { Signer } from "ethers";
```

Comment:
see if can reuse any utils at oa level -> https://github.com/Open-Attestation/open-attestation/blob/master/src/shared/utils/guard.ts#L17

Reply:
the utils only support checking of wrapped and signed documents. it doesn't check if the docs is unwrapped documents, shall I add one in oa utils ?

Reply:
Updated, change to isRawV3Document in OA utils